packages/components/psammead-media-indicator/src/index.jsx

```diff
@@ -38,19 +38,40 @@ const TimeDuration = styled.time`
   margin: 0 ${GEL_SPACING_HLF};
 `;
 
-const MediaIndicator = ({ datetime, duration, type, topStory, service }) => (
-  <MediaIndicatorWrapper
-    aria-hidden="true"
-    topStory={topStory}
-    service={service}
-  >
-    <FlexWrapper>
-      {mediaIcons[type]}
-      {duration && datetime && (
-        <TimeDuration dateTime={datetime}>{duration}</TimeDuration>
-      )}
-    </FlexWrapper>
-  </MediaIndicatorWrapper>
+const IndexAlsosMediaIndicator = styled.span.attrs({ 'aria-hidden': 'true' })`
+  & > svg {
+    margin: 0;
+  }
+`;
+
+const MediaIndicator = ({
+  datetime,
+  duration,
+  type,
+  topStory,
+  service,
+  indexAlsos,
+}) => (
+  <Fragment>
+    {indexAlsos ? (
+      <IndexAlsosMediaIndicator aria-hidden="true" service={service}>
```

Comment:
You already set the `arria-hidden` attribute to the span. And you also don't need to pass `service` to `IndexAlsosMediaIndicator`